invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/collections/menu.overrides

```diff
@@ -222,6 +222,8 @@
       z-index: 100;
       padding: 2rem 1.5rem 1.5rem 1.5rem;
       overflow-y: scroll;
+      overflow-x: clip;
```

Comment:
The x overflow seems to be caused by the `rel-mr-1` on the search bar element
<img width="800" alt="Screenshot 2023-09-13 at 17 35 45" src="https://github.com/inveniosoftware/invenio-app-rdm/assets/21052053/de00fe04-231f-400d-9df9-e30aa004269f">
Instead of clipping the overflow (risky business), can we maybe remove that class from the element, and rather add the margin to the css like this: 

```
.search-bar {
      margin-right: 1rem;

      @media screen and (max-width: @largestTabletScreen) {
        margin-right: 0;
      }
    }
```

Reply:
Right! Thank you very much!